lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -1,33 +1,51 @@
 (function() {
-  var self = this;
-  var reporter = window.ZendeskReporter ? new window.ZendeskReporter() : {};
-
-  // This is just defining a wrapped version of jQuery which simply makes a
-  // pass through call to global window.jQuery and adds logging information
-  // about the origin of the page to Datadog.
-  function wrapped$() {
-    reporter.increment && reporter.increment(
-      'app_framework.app_scope.violated',
-      1,
-      ['origin:' + self.location.hostname]
-    );
-    return self.$.apply(null, arguments);
-  }
-
-  // Adding the wrapped version of jQeury to a clone of window object, which
-  // will be used to bind the app's scope.
-  var wrappedWindow = this.$.extend(true, {}, self, { $: wrapped$ });
-  wrappedWindow.window = wrappedWindow;
-  wrappedWindow.top = wrappedWindow;
+  <% if apps_zaf_naughty_v1_logging %>
+    var self = this;
+    var reporter = window.ZendeskReporter ? new window.ZendeskReporter() : {};
+
+    function getWrappedWindow(appName) {
+      // This is just defining a wrapped version of jQuery which simply makes a
+      // pass through call to global window.jQuery and adds logging information
+      // about the origin of the page to Datadog.
+      function wrapped$(appName) {
+        var tags = ['origin:' + self.location.hostname];
+        if (this.__app_name) {
```

Comment:
How does `this.__app_name` work here? Can we just use `appName` directly?

Reply:
Yes, this is leftover from some previous code change, forgot to remove it.

Reply:
Removed it! 👍